web/src/lib/stores/preferences.store.ts

```diff
@@ -96,3 +96,5 @@ export const albumViewSettings = persisted<AlbumViewSettings>('album-view-settin
   sortDesc: true,
   view: AlbumViewMode.Cover,
 });
+
+export const showDeleteModal = persisted<boolean>('delete-confirm-dialog', true, {});
```

Comment:
This seems fine for me. It can be _per client_.

Reply:
Should we add a toggle in the user settings ?

Reply:
Sureeeeeee